### PR TITLE
add MDAnalysisData package

### PR DESCRIPTION
**Pull request by kain88-de**
@orbeckst 
@richardjgowers 

Do you agree to maintain this package?

### CODE REVIEW

**Comment by orbeckst** [comment]
Yes, I agree.

**Comment by orbeckst** [comment]
Do you also need to add an explicit LICENSE file?

File: recipes/MDAnalysisData/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{% set name = "MDAnalysisData" %}
+{% set version = "0.3" %}
```
Comment (orbeckst) [inline recipes/MDAnalysisData/meta.yaml]: Should this be 0.3.0 – or did I mess up the version on pypi?

File: recipes/MDAnalysisData/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+test:
+  imports:
+    - MDAnalysisData
+
+about:
+  home: https://github.com/MDAnalysis/MDAnalysisData
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
```
Comment (orbeckst) [inline recipes/MDAnalysisData/meta.yaml]: Do you need to explicitly include the file?

File: recipes/MDAnalysisData/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  description: |
+    The MDAnalysisData package unifies access to test and example
+    trajectories that can be used for workshops and extended tests.
+  doc_url: https://www.mdanalysis.org/MDAnalysisData/
+  dev_url: https://github.com/MDAnalysis/MDAnalysisData
+
+extra:
+  recipe-maintainers:
+    - kain88-de
+    - orbeckst
```
Comment (orbeckst) [inline recipes/MDAnalysisData/meta.yaml]: I agree to maintain.

**Comment by kain88-de** [comment]
@conda-forge/staged-recipes can you please have a look at this package. We like to use it for a workshop in November. Thank you!

**Comment by kain88-de** [comment]
Thank you @ocefpaf !

**Pull request merged**